processor/src/decoder/block_stack.rs

```diff
@@ -1,3 +1,5 @@
+use crate::system::ContextId;
+
 use super::{Felt, Vec, Word, ONE, ZERO};
```

Comment:
nit: I would remove the blank line between these two imports.

Reply:
done